packages/@aws-cdk/aws-opensearchservice/rosetta/default.ts-fixture

```diff
@@ -1,12 +1,12 @@
 // Fixture with packages imported, but nothing else
 import { Construct } from 'constructs';
-import { RemovalPolicy, Stack } from '@aws-cdk/core';  
-import * as opensearch from '@aws-cdk/aws-opensearchservice';
+import { Domain, DomainProps, EngineVersion } from '@aws-cdk/aws-opensearchservice';
+import * as core from '@aws-cdk/core'
 import * as lambda from '@aws-cdk/aws-lambda';
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as iam from '@aws-cdk/aws-iam';
 
-class Fixture extends Stack {           
+class Fixture extends core.Stack {
```

Comment:
These updates aren't really necessary.

Reply:
While not strictly necessary, I thought it made sense to update it to follow the documentation recommendations in the contribution guidelines. Specifically `Types from other modules should be qualified:` and `Types from the documented module should be un-qualified:`

Reply:
If you want to change formatting of the documentation, please do it as a separate PR. Large PRs with changes that get lost in formatting changes will not receive the same level of detailed feedback because it's harder to do with so much extra noise. We certainly don't mind better documentation!

Reply:
Alright, I'll split it out into a new PR, though this does mean that the documentation build will continue to fail (it was failing before I made any changes due inconsistent use of "Domain" vs "opensearch.Domain" added in previous PRs). 

That being said, in advance I'm just wondering about this: "Rosetta files, however, should not be manually touched". How is it expected to adjust how imports are represented in the documentation without adjusting the Rosetta file? I'm new to this, so may be missing something, but seems like this is necessary?

Reply:
If you're adding new imports it's fine, but the formatting we use for importing from core is consistent across all packages. No lie, I don't know why that's our convention and why we do it that way, but it's pretty consistent and we should keep it that way.

Reply:
Also, there's a lot of stuff that we autogenerate in various processes through scripts and JSII. To your point about build failures from previous documentation inconsistency, if it's small, it's fine to include with other stuff but in general I'd just open the PR to make that fix first and then do the code change in a subsequent PR. Doc changes usually get merged pretty fast.

Reply:
Makes sense -- thanks for the feedback!

Reply:
So, yes, you can actually update the rosetta file, but you shouldn't change the style of imports. I misspoke earlier when I said they shouldn't be touched at all.